src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -326,7 +335,7 @@ public SnapshotBuilderGetConceptHierarchyResponse getConceptHierarchy(
 
     Map<Integer, SnapshotBuilderParentConcept> parents = new HashMap<>();
     runSnapshotBuilderQuery(
-            query, snapshot, userRequest, ParentQueryResult::new, ParentQueryResult::new)
+            query, snapshot, userRequest, Map.of(), ParentQueryResult::new, ParentQueryResult::new)
```

Comment:
There are a few cases where `Map.of()` is passed. You could instead provide an overload of `runSnapshotBuilderQuery()` which accepts fewer parameters and handles this case for the callers.